src/services/events/events-router.ts

```diff
@@ -0,0 +1,22 @@
+import { Router } from "express";
+import { Database } from "../../database";
+import { StatusCodes } from "http-status-codes";
+import { createId } from "@paralleldrive/cuid2";
+import { Event, EventInfo } from "./events-schema";
+
+const authRouter = Router();
```

Comment:
rename to eventRouter
```suggestion
const eventRouter = Router();
```